src/AdminDashboard/Api/Glpi.js

```diff
@@ -0,0 +1,28 @@
+import 'isomorphic-fetch'
```

Comment:
This lib was updated 3 years ago 😛

Reply:
isomorphic-fetch is implimentation of fetch for node.js and browser, built on top of fetch polyfill, but We could use https://github.com/github/fetch

Reply:
Yes please